extensions/azdata/src/typings/azdata-ext.d.ts

```diff
@@ -147,6 +147,20 @@ declare module 'azdata-ext' {
 		}
 	}
 
+	export interface PostgresServerShowResult {
```

Comment:
Remove - appears to be duplicate of below

Reply:
yeah - looks like an auto-merge issue